src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs

```diff
@@ -104,6 +104,7 @@ internal static class EnforceOnBuildValues
         public const EnforceOnBuild ConsecutiveBracePlacement = /*IDE2002*/ EnforceOnBuild.WhenExplicitlyEnabled;
         public const EnforceOnBuild ConsecutiveStatementPlacement = /*IDE2003*/ EnforceOnBuild.WhenExplicitlyEnabled;
         public const EnforceOnBuild ConstructorInitializerPlacement = /*IDE2004*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild UseUTF8StringLiteral = /*IDE0230*/ EnforceOnBuild.WhenExplicitlyEnabled;
```

Comment:
These used to be ordered by the diagnostic id. Consider moving after `ForEachCast` (IDE0220)